stat/transport.go

```diff
@@ -83,7 +83,10 @@ func encodeError(ctx context.Context, err error, w http.ResponseWriter) {
 func encodeResponse(ctx context.Context, w http.ResponseWriter, response interface{}) (err error) {
 	resp := response.(*common.XmidtResponse)
 
-	w.Header().Set("Content-Type", "application/json")
+	if resp.Code == http.StatusOK {
```

Comment:
It's probably safest to do `w.Header().Del("Content-Type")` for any case in which you don't want a `Content-Type` header.  That ensures that middleware won't unexpectedly clobber your logic.